seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -5,17 +5,58 @@
   "helpRoom": "Help",
   "challenges": [
     {
+      "id": "587d7dac367417b2b2516zx5",
+      "title": "Basic Data Structures: Introduction to Arrays",
+      "description": [
+        [
+          "",
+          "",
+          "<dfn>Arrays</dfn> are one of JavaScript's most fundamental built-in data structures, and probably the data structure that you will work with most often throughout the course of the freeCodeCamp curriculum. The array is by no means unique to JavaScript, however &mdash; in fact, it’s probably safe to say that arrays are utilized by almost every complex program... ever.<br><br>In computer science, an array is a linear collection of elements characterized by the fact that each element has a unique <dfn>index</dfn>, or address, that can be computed and used to look up an element's position at run-time.",
+          ""
+        ],
+        [
+          "",
+          "",
+          "In Javascript, arrays are written as comma separated lists and are enclosed in brackets <code>[...]</code>. They can be any length, and store any type of data supported by JavaScript.<br><br>Arrays can sometimes be simple and serve very basic functionalities, but they can also be complex and very powerful - all of which depends on how the programmer chooses to utilize them<br><br>From this point forward, whenever we refer to the term <dfn>array</dfn>, it will be strictly within the context of JavaScript.",
```

Comment:
I think we should change this: `<code>[...]</code>` to either `<code>[a, b, c]</code>` or `<code>[element1, element2, element3]</code> to prevent confusion with the spread operator.